bot/cogs/cyber.py

```diff
@@ -369,6 +374,20 @@ def suffix_from_number(num):
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
 
+        # Level Function Regex.
+        match = self.game_level_regex.match(message.content)
+        if match and message.channel.id in [HQ_ID, MOON_ID, FORENSICS_ID]:
```

Comment:
This is great; could this be incorporated into the default behavior for the regular level command?

Reply:
I can implement this in a separate PR.